src/actors/upgrade/network_status/actors/netifacescanner/actor.py

```diff
@@ -0,0 +1,125 @@
+from leapp.actors import Actor
+from leapp.models import IfaceResult, IfacesInfo
+from leapp.tags import ScanTag
+import subprocess
+import os
+import re
+import sys
+
+
+class NetIfaceScanner(Actor):
+    name = 'net_iface_scanner'
+    description = 'No description has been provided for the net_iface_scanner actor.'
+    consumes = ()
+    produces = (IfaceResult,)
+    tags = (ScanTag,)
+
+    def process(self):
+        iface_info = self.return_ifs_info()
+        self.produce(iface_info)
+        self.log.info("Finished scanning network interfaces")
+        pass
+
+    def get_ifaces_names(self, sys_command):
+        ifaces = []
+        try:
+            dirs = subprocess.check_output((sys_command),
+                                           stderr=subprocess.STDOUT)
+        except subprocess.CalledProcessError as command_failed:
+            self.log.warning("Could not get the interface names" + command_failed.output)
+            sys.exit(1)
+        dirs = dirs.splitlines()
+        for direct in dirs:
+            ifacelist = os.listdir(direct)
+            for iface in ifacelist:
+                ifaces.append(iface)
+                ifaces = sorted(ifaces)
+        return ifaces
+
+    def get_net_driver(self, iface, devnull):
```

Comment:
Why using devnull? The parameter even the use loookd useless and in future would be great to see err msg in case something will be wrong (I guess it will not be in this case..). In case you want to avoid erroutput, use rather something like:
```
output, dummy_stderr = subprocess.Popen(command,
                                       stdout=subprocess.PIPE,
                                       stderr=subprocess.PIPE)
```
And then just ignore the dummy_stderr variable until someone will find use for it.